postgres/datadog_checks/postgres/postgres.py

```diff
@@ -663,15 +663,16 @@ def _collect_stats(self, key, db, instance_tags, relations, custom_metrics, func
             self.log.error("Connection error: %s" % str(e))
             raise ShouldRestartException
 
-    def _get_service_check_tags(self, host, port, dbname):
+    def _get_service_check_tags(self, host, port, dbname, tags):
```

Comment:
We shouldn't duplicate tags. Here the `db` tag is duplicate on every service check.
This is because the check (line 777) add the database name to the tags and we add it again in the in `service_check_tags`.

Also tiny detail: we should dedup tags in case the user set one tag multiple times.